modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IO.kt

```diff
@@ -804,7 +804,7 @@ sealed class IO<out A> : IOOf<A> {
    * @see [unsafeRunAsyncCancellable] to run in a non-referential transparent manner.
    */
   fun runAsyncCancellable(onCancel: OnCancel = Silent, cb: (Either<Throwable, A>) -> IOOf<Unit>): IO<Disposable> =
-    async { ccb ->
+    Async { _, ccb ->
```

Comment:
@pakoito this is the fix, sorry for the noise around 😅

Reply:
Gotcha. It may have not surfaced before we did `raceN`. Or maybe it's because before we could have the case of returning a value twice or something.